lib/collnew/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "collnew"
+version = "1.0.0"
+authors = ["Vector Contributors <vector@timber.io>"]
+edition = "2018"
+description = "Macros for creating new Collection"
+publish = false
+license = "MIT"
```

Comment:
Is this right? The top-level `Cargo.toml` says `license = "MPL-2.0"`

Reply:
Thanks :+1: 
Fixed.